hail/src/main/scala/is/hail/io/vcf/LoadVCF.scala

```diff
@@ -914,7 +914,7 @@ final class VCFLine(val line: String, arrayElementsRequired: Boolean) {
       if (infoType.hasField(key)) {
         rvb.setFieldIndex(infoType.fieldIdx(key))
         if (infoFlagFieldNames.contains(key)) {
-          if (line(pos) == '=') {
+          if (pos != line.length && line(pos) == '=') {
```

Comment:
should be `!(endField(pos)` I think?

Reply:
`endField` and `pos != line.length` are about as clear in my mind, and the second won't check needlessly for the presence of a tab.

Reply:
OK, fair.